### PR TITLE
Update node-termination-handler image version

### PR DESCRIPTION
**Pull request by yuwenma**
Related issue #13

### CODE REVIEW

File: deploy/k8s.yaml [hunk 1]
```diff
@@ -46,10 +46,10 @@ spec:
                 operator: Exists
       containers:
         # TODO: Update to an official image once the build is automated via GCB.
-      - image: k8s.gcr.io/gke-node-termination-handler@sha256:e08ca863a547754fa7b75064bdad04f04cbef86c7b0a181ecc7304e747623181
+      - image: k8s.gcr.io/gke-node-termination-handler@sha256:aca12d17b222dfed755e28a44d92721e477915fb73211d0a0f8925a1fa847cca 
         name: node-termination-handler
         command: ["./node-termination-handler"]
-        args: ["--logtostderr", "--exclude-pods=$(POD_NAME):$(POD_NAMESPACE)", "-v=10"]
+        args: ["--logtostderr", "--exclude-pods=$(POD_NAME):$(POD_NAMESPACE)", "-v=10", "-taint=blah:blah:blah"]
```
Comment (mindprince) [inline deploy/k8s.yaml]: I think you need actual values here. I don't know what they are though. @vishh would know.

Comment (vishh) [reply deploy/k8s.yaml]: Please use this value: `--taint=cloud.google.com/impending-node-termination::NoSchedule`

**Comment by vishh** [comment]
Can you also update the [Dockerfiles](https://github.com/GoogleCloudPlatform/k8s-node-termination-handler/blob/master/Dockerfile.build) to build off of the new distroless base image? Otherwise a future image push might break your change.

File: Dockerfile.build [hunk 2]
```diff
@@ -6,7 +6,7 @@ COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -tags netgo -o node-termination-handler && go test ./...
 
 # final stage
-FROM alpine
+FROM gcr.io/distroless/static:latest
 WORKDIR /app
 COPY --from=build-env /go/src/github.com/GoogleCloudPlatform/k8s-node-termination-handler/node-termination-handler /app/
-ENTRYPOINT ./node-termination-handler
\ No newline at end of file
+ENTRYPOINT ./node-termination-handler
```
Comment (mindprince) [inline Dockerfile.build]: This needs to be:
```
ENTRYPOINT ["./node-termination-handler"]
```

**Comment by yuwenma** [comment]
Gentle ping.

**Pull request merged**